src/invoices/InvoiceDetails/VoucherInformation/VoucherInformationContainer.js

```diff
@@ -43,10 +65,16 @@ const VoucherInformationContainer = ({ invoiceId, mutator, resources }) => {
   }
 
   return (
-    <VoucherInformation
-      voucher={voucher}
-      voucherLines={voucherLines}
-    />
+    <Accordion
```

Comment:
are you sure this component should have accordion?

Reply:
I need to use `displayWhenOpen` from accordion (it's a button `View voucher`)

Reply:
why not to have this button outside? this component can't be reused without accordion set

Reply:
We don't need to reuse this container. A reusable component is `<VoucherInformation />` inside this container